validation-test/stdlib/StringUTF8.swift

```diff
@@ -47,8 +48,33 @@ var strings: Array<String> = [
   "日",
 ]
 
+var utf16ByteSequences = utf16Tests.flatMap { $0.value }.map { $0.encoded }
+
 let kCFStringEncodingASCII: UInt32 = 0x0600
 
+private func testForeignContiguous(slowString: NSSlowString, string: String) {
```

Comment:
This is just the bit below pulled out so I can reuse it for the second set of data